client/dashboard/data/index.ts

```diff
@@ -97,51 +99,34 @@ export const fetchSiteMediaStorage = async ( id: string ): Promise< MediaStorage
 };
 
 export const fetchSiteMonitorUptime = async (
-	id: string
+	site: Site
```

Comment:
I'm not sure I like the fact that a simple fetch function gets a site object

Reply:
Why?

Reply:
Can we do site ID + a simple setting?

Reply:
Also it seems that it doesn't really need it either. That Jetpack check does not really belong here

Reply:
Ok, wait, I have a better idea, let's check before calling

Reply:
Better? 538c098eb90949881e6bc106aaa11d94abee4282